builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1091,6 +1104,10 @@ def loadBuildModules(buildModuleUrl){
 
 		sonarModule = load "sonar-module.groovy"
 		echo "Sonar module loaded successfully."
+		
+		apigeeModule = load "apigee-module.groovy"
```

Comment:
Load this only when APIGEE extension is enabled by admin (when APIGEE specific configurations exist in config).

Reply:
Updated for next commit.